test/ConfirmationButton_test.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint func-names: "off" */
+
+import assert from "assert";
+import React from "react";
+import sinon from "sinon";
+import {mount, shallow} from "enzyme";
+import {Button, ModalButton, ConfirmationButton} from "../src";
+
+describe("ConfirmationButton", () => {
+  it("renders a modal with confirm and cancel buttons", () => {
+    const confirmationButton = shallow(<ConfirmationButton value="Confirm pl0x" modalTitle="A title">
+      <p>Hello</p>
+    </ConfirmationButton>);
+    assert.equal(confirmationButton.find(ModalButton).length, 1);
+    assert(!!confirmationButton.find(ModalButton).props().children.find(
+      (child) => {
```

Comment:
could we express this test with selectors like you do in later tests? for instance you could do something like
```js
confirmationButton.find("button").simulate("click");
const cancelButton = confirmationButton.find("button.ConfirmationButton--dialog-buttons[value='Cancel'])");
const confirmButton = confirmationButton.find("button.ConfirmationButton--dialog-buttons[value='Confirm'])");
assert(cancelButton.length, 1);
assert(confirmButton.length, 1);
```